servers/jax-rs-tests/src/main/java/org/projectnessie/jaxrs/AbstractRestMergeTransplant.java

```diff
@@ -50,6 +56,12 @@
 /** See {@link AbstractTestRest} for details about and reason for the inheritance model. */
 public abstract class AbstractRestMergeTransplant extends AbstractRestInvalid {
 
+  static {
+    ConfigurationProvider.CONFIGURATION_PROVIDER
```

Comment:
Can you move it to a file similar to `versioned/tests/src/main/resources/META-INF/services/org.assertj.core.configuration.Configuration`

Reply:
I removed this code block completely. It is not needed under the new versioned test extensions.